src/test/java/seedu/module/logic/commands/DeleteTagCommandTest.java

```diff
@@ -0,0 +1,119 @@
+package seedu.module.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.module.logic.commands.CommandTestUtil.VALID_TAG_HIGH;
+import static seedu.module.logic.commands.CommandTestUtil.VALID_TAG_LOW;
+import static seedu.module.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.module.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.module.logic.commands.CommandTestUtil.showTaskAtIndex;
+import static seedu.module.testutil.TypicalIndexes.INDEX_FIRST_TASK;
+import static seedu.module.testutil.TypicalIndexes.INDEX_SECOND_TASK;
+import static seedu.module.testutil.TypicalTasks.getTypicalModuleBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.module.commons.core.Messages;
+import seedu.module.commons.core.index.Index;
+import seedu.module.model.Model;
+import seedu.module.model.ModelManager;
+import seedu.module.model.ModuleBook;
+import seedu.module.model.UserPrefs;
+import seedu.module.model.tag.Tag;
+import seedu.module.model.task.Task;
+import seedu.module.testutil.TaskBuilder;
+
+class DeleteTagCommandTest {
+
+    private final Model model = new ModelManager(getTypicalModuleBook(), new UserPrefs());
+
```

Comment:
I think one more test for "tag does not exist"?

Reply:
sure!